deployment/src/strongmind_deployment/rails.py

```diff
@@ -112,3 +117,24 @@ def get_database_url(self):
                              '@',
                              self.rds_serverless_cluster.endpoint,
                              ':5432/app')
+
+    def dns(self, name):
+        if self.env_name != "prod":
+            name = f"{self.env_name}-{name}"
+        domain = 'strongmind.com'
+        zone_id = self.kwargs.get('zone_id')
+        if not zone_id:  # pragma: no cover
+            zone_id = get_zone(account_id='8232ad8254d56191adf53b86920459fa', name=domain).zone_id
```

Comment:
I'd like to eventually find a better way to deal with these, but I don't yet have a strong suggestion about what method to choose.